crates/bevy_pbr/src/render/mesh_view_bindings.wgsl

```diff
@@ -47,29 +47,32 @@ var<uniform> globals: Globals;
 var<uniform> fog: Fog;
 
 @group(0) @binding(11)
-var environment_map_diffuse: texture_cube<f32>;
+var screen_space_ambient_occlusion_texture: texture_2d<f32>;
```

Comment:
It's not particularly necessary, but moving this after the dt_lut_sampler bind group would reduce a bit of noise in the PR around binding index changing.

Reply:
Yeah not a huge deal either way (and normally I'd agree), but I think the bigger diff here is worth it because it makes the code slightly simpler conceptually: `main_block_of_uniforms_with_ssao -> array_of_env_map_bindings -> array_of_lut_bindings` vs `main_block_of_uniforms -> array_of_env_map_bindings -> array_of_lut_bindings -> single_ssao_binding`